package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screwdriver-data-schema",
-  "version": "13.0.0",
+  "version": "12.3.0",
```

Comment:
I think because it's still WIP, we're in a inconsistent state.  I'd revert the change, keep at 13, and open a PR to revert the revert when ready.